src/inmanta/protocol/common.py

```diff
@@ -878,6 +885,20 @@ def _encode_dict_for_get(
         sub_dict = {f"{query_param_name}.{key}": value for key, value in query_param_value.items()}
         return sub_dict
 
+    def get_openapi_parameter_type_for(self, param_name: str) -> Optional[openapi_model.ParameterType]:
```

Comment:
I just noticed you use this in your other PR. I think it should be marked with `@stable_api`.

Reply:
I added them